### PR TITLE
Faqs added in About Activity

### LINKED ISSUE #1246: Add FAQs page in Commons App
**Summary:** 

We need to add a FAQs page in App.We can choose some FAQs from the website.
Link to FAQs - https://commons.wikimedia.org/wiki/Commons:FAQ#What_is_this_site_about?

**Steps to reproduce:** 

Add a new Activity /Fragment for FAQs Page.

**Expected behavior:** 

There should be a page for FAQs.

**Observed behavior:** 

There is no FAQ page in the app now.

**Screen-shots:** 
UI can be like this.
<table>
<tr>
<td>

screenshot_20180303-155305

</td>
<td>

screenshot_20180303-155312

</td>
</tr>
</table>

or we can add a link in about page of the App.

### PR DESCRIPTION
**Pull request by ujjwalagrawal17**
## Description

Fixes #1246 

Changes:
1. Textview added in activity_about.xml for showing "Frequently Asked Questions"
2. Onclick of about_faq added in AboutActivity.java which opens [FAQ](https://github.com/commons-app/apps-android-commons/wiki/Frequently-Asked-Questions)

## Screenshots showing what changed
 
<table>
<tr>
<td>

screenshot_20180304-125514

</td>
<td>

screenshot_20180304-125357

</td>
</tr>
</table>

### CODE REVIEW

**Comment by ujjwalagrawal17** [comment]
I will update FAQs in wiki now.

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -240,4 +240,5 @@
   <string name="notifications_mention">%1$s mentioned you on %2$s.</string>
   <string name="toggle_view_button">Toggle view</string>
   <string name="about_rate_us"><u>Rate Us</u></string>
+  <string name="about_faq"><u>Frequently Asked Questions</u></string>
```
Comment (maskaravivek) [inline app/src/main/res/values/strings.xml]: <u> tag might have issues with translation. Please check #1258.

Comment (nicolas-raoul) [reply app/src/main/res/values/strings.xml]: I think underlines should be added afterwards by the app, they do not need to be translated, right?

Comment (ujjwalagrawal17) [reply app/src/main/res/values/strings.xml]: I found this method most suitable https://stackoverflow.com/questions/2394935/can-i-underline-text-in-an-android-layout .

Comment (ujjwalagrawal17) [reply app/src/main/res/values/strings.xml]: Should I change it using Java code?

Comment (ujjwalagrawal17) [reply app/src/main/res/values/strings.xml]: I got it.

Comment (maskaravivek) [reply app/src/main/res/values/strings.xml]: @ujjwalagrawal17 Can you briefly explain about not changing it in the java code instead?

Comment (ujjwalagrawal17) [reply app/src/main/res/values/strings.xml]: We can change it using java code also but I thought if we can do it in XML it would be simpler.

**Comment by maskaravivek** [comment]
Doesn't work for me. 

device-2018-03-07-065012

**Comment by ujjwalagrawal17** [comment]
I had changed it using Java now.
 ```
SpannableString content = new SpannableString(getString(R.string.about_faq));
 content.setSpan(new UnderlineSpan(), 0, content.length(), 0);
 faqText.setText(content);
```
**Screenshot**

screenshot_20180304-125514

**Comment by ujjwalagrawal17** [comment]
@maskaravivek Do i need to change anything else ?

**Comment by maskaravivek** [comment]
Tested on beta build. Works fine for me. Going ahead and merging it. Thanks for the contribution @ujjwalagrawal17. :)

**Pull request merged**